privateGPT.py

```diff
@@ -16,23 +16,53 @@
 model_type = os.environ.get('MODEL_TYPE')
 model_path = os.environ.get('MODEL_PATH')
 model_n_ctx = os.environ.get('MODEL_N_CTX')
+is_gpu_enabled = os.environ.get('IS_GPU_ENABLED')
```

Comment:
Better make it boolean here and avoid the  `== "True"` comparison in the rest of the code

`is_gpu_enabled = (os.environ.get('IS_GPU_ENABLED', 'False').lower() == 'true')` 

Supports True or true as valid env var values for True